zcash_client_backend/proto/service.proto

```diff
@@ -73,6 +73,7 @@ message LightdInfo {
     uint64 estimatedHeight = 12;        // less than tip height if zcashd is syncing
     string zcashdBuild = 13;            // example: "v4.1.1-877212414"
     string zcashdSubversion = 14;       // example: "/MagicBean:4.1.1/"
+    string donationAddress = 15;        // Zcash donation UA address
```

Comment:
The problem with this approach is s that we don't validate the address that is being set up.